pkg/addons/default/aws_node_generate.go

```diff
@@ -1,3 +1,3 @@
 package defaultaddons
 
-//go:generate curl --silent --location https://github.com/aws/amazon-vpc-cni-k8s/blob/61c8d18c0e097c0b7e8477e1afc61d6f2601295d/config/v1.6/aws-k8s-cni.yaml?raw=1 --output assets/aws-node.yaml
```

Comment:
@sayboras can you update using current HEAD? some lines have been updated

Reply:
@michaelbeaumont Just want to confirm again that you want to lock it to HEAD instead of v1.6.1. It's not consistent with the convention we have right now https://docs.aws.amazon.com/eks/latest/userguide/cni-upgrades.html

Reply:
ah sorry i meant not track it to HEAD per se, just to the commit which is currently HEAD

Reply:
@sayboras [according to the docs](https://docs.aws.amazon.com/eks/latest/userguide/cni-upgrades.html), we should use:
https://github.com/aws/amazon-vpc-cni-k8s/commits/release-1.6 when updating

because at the moment we're using a static asset, can you pin it to the current version of that branch?

Reply:
Thanks for your input, I have updated the PR accordingly.